plugins/toolkit/jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/java/JavaLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -103,7 +107,9 @@ class JavaLocalLambdaRunConfigurationIntegrationTest(private val runtime: Lambda
             project = projectRule.project,
             runtime = runtime.toSdkRuntime(),
             input = "\"Hello World\"",
-            credentialsProviderId = mockId
+            credentialsProviderId = mockId,
+            environmentVariables = mutableMapOf("GRADLE_USER_HOME" to gradleUserHomeDir),
+            samOptions = SamOptions(buildInContainer = true)
```

Comment:
As expected, making it run in a containerized environment doesn't require env vars to be set The test case passed since container has its own grade user home dir.